src/apikey/index.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import { SDKAuthHelper } from "../common/types";
+import { AwsCredentialIdentityProvider, SDKAuthHelper } from "../common/types";
```

Comment:
Since amazon-location-utilities-auth-helper-js is going to depend on `aws-sdk` anyways, I think it is OK to just use `AwsCredentialIdentityProvider` from `@aws-sdk/types` instead of re-exporting.